SproxydKeysScan/DuplicateKeysIngestion.js

```diff
@@ -63,7 +63,7 @@ class RaftJournalReader {
      */
     setBegin(cb) {
         if (this.begin) {
-            return process.nextTick(cb);
+            return cb();
```

Comment:
Is this revert intentional?

Reply:
jest tests hung with process.nextTick(cb). I think process.next tick was waiting for the unit tests to resolve. I didn't investigate further, seemed returning cb() was ok. @jonathan-gramain